config/secrets.yml.example

```diff
@@ -1,9 +1,18 @@
 development:
   secret_token: SOMETHING_RANDOM_HERE
   secret_key_base: SOMETHING_RANDOM_HERE
+  facebook_auth:
+    app_id: YOUR_FACEBOOK_LOGIN_APP_ID
+    app_secret: YOUR_FACEBOOK_LOGIN_APP_SECRET
 test:
   secret_token: SOMETHING_RANDOM_HERE
   secret_key_base: SOMETHING_RANDOM_HERE
+  facebook_auth:
+    app_id: YOUR_FACEBOOK_LOGIN_APP_ID
+    app_secret: YOUR_FACEBOOK_LOGIN_APP_SECRET
 production:
   secret_token: SOMETHING_RANDOM_HERE
   secret_key_base: SOMETHING_RANDOM_HERE
+  facebook_auth:
+    app_id: YOUR_FACEBOOK_LOGIN_APP_ID
+    app_secret: YOUR_FACEBOOK_LOGIN_APP_SECRET
```

Comment:
We're trying to keep as much as possible in the Settings. You can add settings to the `config/settings.yml` file and then display them in the [Settings screen](https://github.com/churchio/onebody/blob/master/app/views/administration/settings/index.html.haml) so the user can easily edit them via the UI.